pkg/objcache/objcache.go

```diff
@@ -27,18 +27,20 @@ import (
 	"time"
 )
 
-// NoExpiry represents caches to be permanent and can only be deleted.
-var NoExpiry = time.Duration(0)
+const (
+	// NoExpiry represents caches to be permanent and can only be deleted.
+	NoExpiry = time.Duration(0)
 
-// DefaultExpiry represents default time duration value when individual entries will be expired.
-var DefaultExpiry = time.Duration(72 * time.Hour) // 72hrs.
+	// DefaultExpiry represents three days time duration when individual entries will be expired.
+	DefaultExpiry = time.Duration(3 * 24 * time.Hour)
 
-// DefaultBufferRatio represents default ratio used to calculate the
-// individual cache entry buffer size.
-var DefaultBufferRatio = uint64(10)
+	// defaultBufferRatio represents default ratio used to calculate the
+	// individual cache entry buffer size.
+	defaultBufferRatio = uint64(10)
 
-// DefaultGCPercent represents default garbage collection target percentage.
-var DefaultGCPercent = 20
+	// defaultGCPercent represents default garbage collection target percentage.
+	defaultGCPercent = 20
```

Comment:
I see `New()` is using these values.  If any setter is called after `New()` there is no effect.
When are they getting used other than `New()`?

Problem with these globals are we should use like below
```go
cache1 := New(...)
// Modify globals
cache2 := New(...)

```

This is not a good way of doing it.  Also it is problematic in concurrent programming.